core/src/main/scala/zio/config/ConfigSource.scala

```diff
@@ -33,6 +33,29 @@ object ConfigSource {
   def empty[K, V]: ConfigSource[K, V] =
     ConfigSource(_ => PropertyTree.empty, Nil)
 
+  /**
+   * Forming configuration from command line arguments, eg `List(--param1=xxxx, --param2=yyyy)`
+   *
+   * Pack all of the command-line arguments into multiple property lists. Using PropertyTree.mergeAll, merge a
+   * bunch of command line options into the smallest possible set of property trees, and then use those
+   * property trees to perform lookup.
+   *
+   * This is an experimental implementation for handling of key/value switches, and is not a
```

Comment:
Could you have this comment in both `fromArgs` method in the project.

Let's say `EXPERIMENTA` in the first line of  the comment

Reply:
Done